File: Material/TransitSoft/TransitSoftPersistance/DAO/CapturaDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TransitSoft.Model;

namespace TransitSoft.DAO
{
    public interface CapturaDAO
    {
        List<Captura> LeerTodos();
        bool Actualizar(Captura captura);
        int? Insertar(Captura captura);
        int? Modificar(Captura captura);
        int? Eliminar(int id);
        Captura ObtenerPorId(int id);
    }
}

File: EX2/EX2/Material/TransitSoft_ConexionMultiple - REST/TransitSoftPersistencia/src/main/java/pe/edu/pucp/transitsoft/dao/CapturaDAO.java
package pe.edu.pucp.transitsoft.dao;

import java.util.List;
import pe.edu.pucp.transitsoft.modelo.Captura;

// TODO: Definir Interfaz CapturaDAO
public interface CapturaDAO {
    public List<Captura> leerTodos();
    public boolean actualizar(Captura captura);
    public Integer insertar(Captura captura);
    public Integer modificar(Captura captura) ;
    public Integer eliminar(int id);
    public Captura obtenerPorId(int id);
}
